composer.json

```diff
@@ -8,18 +8,35 @@
         "php": ">=7.2",
         "ext-intl": "*"
     },
+    "require-dev": {
+        "codeigniter4/codeigniter4": "dev-develop",
+        "phpunit/phpunit": "^8.5"
+    },
+    "config": {
+        "optimize-autoloader": true,
+        "preferred-install": "dist",
+        "sort-packages": true
+    },
     "autoload": {
         "psr-4": {
-            "Translations\\": ""
+            "CodeIgniter\\Language\\": "Language/"
         }
     },
+    "autoload-dev": {
+        "psr-4": {
+            "CodeIgniter\\Language\\": "tests/system/Language/"
+        }
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/codeigniter4/codeigniter4",
```

Comment:
```suggestion
            "url": "https://github.com/codeigniter4/CodeIgniter4",
```

Reply:
Composer writes package names using lowercase, so no matter how you write the repo name it will be parsed as lowercased. Check composer's code for details.